webapp/src/containers/Rhs/rhsHeader.tsx

```diff
@@ -58,22 +60,20 @@ const Header = ({
                             </SvgWrapper>
                         </IconButton>
                     )}
-                    <button
-                        className={`btn btn-primary share-record-btn ${!showFilterIcon && 'share-record-btn__without-filter'}`}
-                        onClick={() => dispatch(setGlobalModalState({modalId: 'shareRecord'}))}
+                    <IconButton
+                        tooltipText='Actions'
+                        onClick={() => setShowMenu(!showMenu)}
+                        extraClass={`${!showFilterIcon && 'margin-left-auto'}`}
```

Comment:
```suggestion
                        extraClass={!showFilterIcon && 'margin-left-auto'}
```

Reply:
getting typescript error on doing this

Reply:
try `extraClass={!showFilterIcon ? 'margin-left-auto' : ''}`